CMakeLists.txt

```diff
@@ -44,22 +44,45 @@ if(APPLE)
   set(CMAKE_CXX_FLAGS "-stdlib=libc++")
 endif()
 
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pipe -O2 -Wall -W -fPIE")
```

Comment:
I am pretty ignorant about all the flags, to be honest. ;)
How I came to this set of flags:
I compiled with CMake, and I got segfaults when starting openscad. Then I compiled with the qmake Makefile, and there were no problems; no segfaults. I then compared the actual flags used by the two Makefiles (qmake & CMake generated), and I added all the ones missing from the qmake Makefile to CMake. I compiled wiht CMake, and there were no more segfaults.

So... I would be fine to reduce this set of flags to a minimum, if you have a good guess about which flags are the essential ones here. My own guess would be pretty crude, and because one build takes at least 20min on my system, I'd rather not test through too many combinations of flags.
I am pretty sure that `-frounding-math` is essential though.

Reply:
I wrote this commit abou ta year ago. I now tested without any of the flags, and with just rounding math, and neither of the produced binaries gives me any segfaults anymore. ;-)
so.. I willl make a commit with `-frounding-math` for GCC and Clang, and forget about al lthe other flags.
thanks, and sorry for causing you all this work!

Reply:
done